pulp_smash/cli.py

```diff
@@ -234,6 +236,18 @@ def run(self, args, **kwargs):
         completed_process = CompletedProcess(args, code, stdout, stderr)
         return self.response_handler(completed_process)
 
+    def sudo(self, args, **kwargs):
+        """Run a command as superuser.
+
+        This method is a wrapper around :meth:`pulp_smash.cli.Client.run`
+        which prior to execution checks if user is root and if not
+        adds ``sudo`` as the first argument of the command.
+        """
+        if args[0] != 'sudo' and not is_root(self.cfg):
```

Comment:
@kersommoura The space there is because the use of `.split()` on a string. As  `run` will accept only a list/tuple it is not a problem, we can replace those calls with just `client.sudo`.

Reply:
@rochacbruno, great.